tests/auto/perfdata/tst_perfdata.cpp

```diff
@@ -214,6 +214,8 @@ void TestPerfData::testFiles_data()
         "vector_static_gcc/perf.data",
         "vector_static_gcc/perf.data.zstd",
         "fork_static_gcc/perf.data.zstd",
+        "disassembler_dwarf/perf.dwarf.data",
```

Comment:
is this new demo really required, can't we just use the existing test binary and perf.data files? what's missing from this patch is actually writing all the new parameters you have added (ideally within each patch, so it's easy to see what they are changing).  see PerfParserTestClient::convertToText

Reply:
> is this new demo really required, can't we just use the existing test binary and perf.data files? 
I am not sure I got what do you mean as 'existing test binary and perf.data files'. 
Which are?

Reply:
`tests/manual/clients/` there are the sources: `vector.cpp` and `fork.c`

in `tests/auto/perfdata/` you'll find three static binaries compiled from these two source files. And `vector_static_gcc` e.g. also shows how the same binary can have multiple `perf.data` files (one with zstd and one without). So eventually you should one with LBR together with whatever patch you are writing to fix something. Ideally, always first add the new test file and then in a follow up patch fix the issue such that we can clearly see the difference it makes to the expected test  files